src/components/VFilters/VFilterChecklist.vue

```diff
@@ -20,40 +20,54 @@
         <VLicense v-if="filterType === 'licenses'" :license="item.code" />
         <template v-else>{{ itemLabel(item) }}</template>
       </VCheckbox>
-      <button
-        v-if="filterType === 'licenses'"
-        :ref="`${item.code}licenseIcon`"
-        :aria-label="$t('browse-page.aria.license-explanation')"
-        class="text-dark-charcoal-70 appearance-none"
-        type="button"
-        @click.stop="toggleLicenseExplanationVisibility(item.code)"
-      >
-        <VIcon :icon-path="helpIcon" />
-      </button>
+
+      <!-- License explanation -->
+      <VPopover v-if="filterType === 'licenses'">
+        <template #trigger="{ a11yProps }">
+          <button
+            v-bind="a11yProps"
+            :ref="`${item.code}licenseIcon`"
```

Comment:
It doesn't look like we're using this ref anywhere anymore:

```suggestion
```

Reply:
Good catch, fixed in b13ea0c.